File: Python/Ex_26.py
v1 = int(input('Type a value: '))

while v1 < 0:
    v1 = int(input('Type a value again. Only positive values: '))

print(f"Printing number {v1} times table...")

for c in range (1, 11):
    print(f"{c} x {v1} = {c * v1}")

File: c/Ex_26.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int v1;

    printf("Type a value: ");
    scanf("%i", &v1);

    while (v1 < 0)
    {
        printf("Type a value again. Only positive values: ");
        scanf("%i", &v1);
    }

    printf("Printing number %i times table... ", v1);

    for (int i = 1; i < 11; i++)
    {
        printf("%i x %i = %i", i, v1, i * v1);
    }
    
    return 0;
}
